srcpkgs/pencil2d/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'pencil2d'
+pkgname=pencil2d
+version=0.6.2
+revision=1
+build_style=qmake
+wrksrc=pencil-${version}
+hostmakedepends="qt5-qmake"
+makedepends="qt5-xmlpatterns-devel qt5-svg-devel qt5-multimedia-devel"
+depends="qt5-multimedia qt5-svg qt5-xmlpatterns hicolor-icon-theme ffmpeg"
```

Comment:
I would expect that they are all getting automatically detected by xbps-src, except `hicolor-icon-theme`. So this can be reduced to `depends="hicolor-icon-theme"`